### ISSUE TITLE
SC2154: Treat parameter expansion using @ same as *

### ISSUE THREAD

**Issue by ChillerDragon** [koalaman/shellcheck#2116]
#### For bugs
- SC2154
- shellcheck version: online 
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2034)
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

x_foo=henlo
x_bar=cruel
x_baz=world

echo "${!x_@}"
echo "${!x_*}"
```

#### Here's what shellcheck currently says:

```
In test.sh line 3:
x_foo=henlo
^---^ SC2034: x_foo appears unused. Verify use (or export if used externally).


In test.sh line 4:
x_bar=cruel
^---^ SC2034: x_bar appears unused. Verify use (or export if used externally).


In test.sh line 5:
x_baz=world
^---^ SC2034: x_baz appears unused. Verify use (or export if used externally).


In test.sh line 7:
echo "${!x_@}"
      ^-----^ SC2154: x_ is referenced but not assigned.
```

#### Here's what I wanted or expected to see:

At least omit the SC2154 since the SC2034 unused warning seems to be a design decision to ignore indirect references.

**Comment by koalaman** [comment]
You're right, of course. This was a simple oversight and has now been fixed. Thanks!
